puppet/modules/kubernetes/manifests/kubelet.pp

```diff
@@ -26,7 +26,7 @@
   $pod_cidr = undef,
   $hostname_override = undef,
   Enum['systemd', 'cgroupfs'] $cgroup_driver =  $::osfamily ? {
-    'RedHat' => 'systemd',
+    'RedHat' => 'cgroupfs',
```

Comment:
can this not just rely on the default if it's now the same?

Reply:
yeah true, will change